packages/flutter/lib/src/material/popup_menu.dart

```diff
@@ -1073,6 +1084,24 @@ class PopupMenuButton<T> extends StatefulWidget {
   /// If this property is null, the default size is 24.0 pixels.
   final double? iconSize;
 
+  /// Optional size constraints for the menu.
+  ///
+  /// When unspecified, defaults to:
+  /// ```dart
+  /// const BoxConstraints(
+  ///   minWidth: _kMenuMinWidth,
+  ///   maxWidth: _kMenuMaxWidth,
+  /// )
+  /// ```
+  /// where [_kMenuMinWidth] is `5.0 * _kMenuWidthStep`, `_kMenuWidthStep` is
```

Comment:
If you put the values they evaluate to into the above, you can eliminate this paragraph.